ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3Extension.java

```diff
@@ -64,10 +64,11 @@
     public static final String NAMESPACE_7_0 = EJB3SubsystemNamespace.EJB3_7_0.getUriString();
     public static final String NAMESPACE_8_0 = EJB3SubsystemNamespace.EJB3_8_0.getUriString();
     public static final String NAMESPACE_9_0 = EJB3SubsystemNamespace.EJB3_9_0.getUriString();
+    public static final String NAMESPACE_10_0 = EJB3SubsystemNamespace.EJB3_10_0.getUriString();
 
     static final PathElement SUBSYSTEM_PATH = PathElement.pathElement(ModelDescriptionConstants.SUBSYSTEM, SUBSYSTEM_NAME);
 
-    static final ModelVersion CURRENT_MODEL_VERSION = EJB3Model.VERSION_9_0_0.getVersion();
+    static final ModelVersion CURRENT_MODEL_VERSION = EJB3Model.VERSION_10_0_0.getVersion();
```

Comment:
This constant duplicates EJB3Model.CURRENT.  I suggest removing this constant and replacing any references with EJB3Model.CURRENT.getVersion().

Reply:
Done.